src/hypertrace/agent/constants.py

```diff
@@ -1,4 +1,9 @@
 '''Agent generic exception message'''
+from autoinstrumentation.src.hypertrace.instrumentation.version import __version__
```

Comment:
I am curious about this, I think we should have the version in the root of the repo instead of this belonging to specific package.

Reply:
yes I agree __version__ should be at root

Reply:
It now is.